tcms/issuetracker/redmine_integration.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+    Helper which facilitate actual communications with JIRA.
```

Comment:
Next time be more careful with copying files around.

Reply:
OK